localstack/utils/server/proxy_server.py

```diff
@@ -90,22 +84,9 @@ def start_ssl_proxy(
     target_ssl=False,
     client_cert_key: Tuple[str, str] = None,
     asynchronous: bool = False,
-    fix_encoding: bool = False,
```

Comment:
This one caught my attention - I traced it back to a chance in -ext we introduced a while ago, in order to be able to deploy this EKS TF sample: https://github.com/terraform-aws-modules/terraform-aws-eks/tree/b876ff95136fbb419cbb33feaa8f354a053047e0/examples/irsa

Found another change that we added a bit later, so the use of the `fix_encoding=True` flag in -ext was removed in the process.

Did a quick check to deploy the TF sample - works with the latest version on `master` (after a small fix in moto), even without the flag. 👍 So, this should be good to be removed!